Formula/age-plugin-yubikey.rb

```diff
@@ -0,0 +1,25 @@
+class AgePluginYubikey < Formula
+  desc "Plugin for encrypting files with age and PIV tokens such as YubiKeys"
+  homepage "https://github.com/str4d/age-plugin-yubikey"
+  url "https://github.com/str4d/age-plugin-yubikey/archive/v0.3.0.tar.gz"
+  sha256 "3dfd7923dcbd7b02d0bce1135ff4ba55a7860d8986d1b3b2d113d9553f439ba9"
+  license "MIT"
+  head "https://github.com/str4d/age-plugin-yubikey.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  uses_from_macos "pcsc-lite"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "pcsc-lite"
```

Comment:
`uses_from_macos "pcsc-lite"` does exactly this

```suggestion
```

Reply:
Done, thanks.